### PR TITLE
Added start script inside package.json using node monitor

### PR DESCRIPTION
**Pull request by NJaryal**

### CODE REVIEW

File: api/routes/news.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    res.status(200).json({
+        message: 'Single News!',
+        newsId: req.params.id,
+        obj: newsObj
+    })
+});
+
+router.post('/', (req, res, next) => {
+    const {error} = validateNews(req.body);
+    if(error) return res.status(400).send(error.details[0].message)
```
Comment (byslavik) [inline api/routes/news.js]: Please use error component here

File: app.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    } else {
+        logger.warn(`Error ${error.status} ` ,new Error(error.message));
+    }        
+    next();
+})
+
+//Routes to handle Request
+app.use('/news', newsRoutes)
+
+app.use((req,res,next)=> {
```
Comment (byslavik) [inline app.js:20]: Do we need that middleware? in case of we have error handler bellow

File: api/routes/news.js [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+        id: 3,
+        name: "Jack"
+    },
+    {
+        id: 4,
+        name: "Kohli"
+    }
+];
+
+const errorGenerator = (text, status)=> {
```
Comment (byslavik) [inline api/routes/news.js]: class CustomError extends Error {
  constructor(text, status){
    super(text)
    
    this.status = status
  }
  
}


Then just do throw new CustomError(text, status)

File: app.js (hunk 2 above)
Comment (NJaryal) [reply app.js:20]: I am using express router so this is how i understood.

Comment (byslavik) [reply app.js:20]: We can remove it

Comment (byslavik) [reply app.js:20]: Or better, lets replace with 

app.get('*', (req,res)=> {
  throw new CustomError('Not Found', 404)
})

File: api/routes/news.js [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+        id: 3,
+        name: "Jack"
+    },
+    {
+        id: 4,
+        name: "Kohli"
+    }
+];
+
+class CustomError extends Error{
```
Comment (byslavik) [inline api/routes/news.js]: Lets move it into separate file

File: api/routes/news.js [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+    constructor(text, status){
+        super(text)
+        this.status = status
+    }
+}
+
+const validateNews = (newsObj)=> {
```
Comment (byslavik) [inline api/routes/news.js]: And that function as well

File: api/routes/news.js (hunk 3 above)
Comment (NJaryal) [reply api/routes/news.js]: Resolved

File: api/routes/news.js (hunk 4 above)
Comment (NJaryal) [reply api/routes/news.js]: Moved

File: api/routes/news.js (hunk 5 above)
Comment (NJaryal) [reply api/routes/news.js]: Moved

File: app.js (hunk 2 above)
Comment (NJaryal) [reply app.js:20]: We should be good now

File: api/routes/news.js (hunk 1 above)
Comment (NJaryal) [reply api/routes/news.js]: Updated

**Pull request merged**